spec/researches/topicCountSpec.js

```diff
@@ -77,4 +77,9 @@ describe( "Test for counting the keyword and synonyms in a text", function() {
 			{ keyword: "", synonyms: "" } );
 		expect( topicCount( mockPaper ).count ).toBe( 0 );
 	} );
+
+	it( "only counts keywords when the onlyKeyword option is set to true", function() {
```

Comment:
Please change to "counts only keywords (and not synonyms) when the onlyKeyword option is set to true".